Portfolio/src/assets/Variables.scss

```diff
@@ -1,11 +1,23 @@
-// $color-pink-primary: #D4ABB2;
-// $color-black-heading: #1e1e1e;
-// $color-white: #ffff;
-
-// $font-family-primary: "Raleway";
-// $font-size-small: 1.5rem;
-
-// $margin-top-small: 5rem;
-// $margin-top-large: 10rem;
+$margin-top-xsmall: 1rem;
+$margin-top-medium: 5rem;
+$margin-top-large: 10rem;
+$margin-zero: 0;
+$margin-medium: 4rem;
```

Comment:
What is the philosophy behind having this margin as "medium" and not small or large? Question regarding application scaling. What would u do if u in the future add 10 rem, 15rem, 25rem, 50rem, 100rem? 4rem would not still be the medium...

Reply:
Fixed